internal/bridge/crunchybridgecluster/crunchybridgecluster_controller.go

```diff
@@ -445,23 +460,23 @@ func (r *CrunchyBridgeClusterReconciler) Reconcile(ctx context.Context, req ctrl
 	// then we will requeue and wait for it to be done.
 	// TODO(crunchybridgecluster): Do we want the operator to interrupt
 	// upgrades created through the GUI/API?
-	if len(crunchybridgecluster.Status.ClusterUpgrade.Operations) != 0 {
+	if len(crunchybridgecluster.Status.OngoingUpgrade) != 0 {
 		return ctrl.Result{RequeueAfter: 3 * time.Minute}, nil
 	}
 
 	// Check if there's an upgrade difference for the three upgradeable fields that hit the upgrade endpoint
 	// Why PostgresVersion and MajorVersion? Because MajorVersion in the Status is sure to be
-	// an int of the major version, whereas Status.Cluster.PostgresVersion might be the ID
-	if (storageVal != crunchybridgecluster.Status.Cluster.Storage) ||
-		crunchybridgecluster.Spec.Plan != crunchybridgecluster.Status.Cluster.Plan ||
-		crunchybridgecluster.Spec.PostgresVersion != crunchybridgecluster.Status.Cluster.MajorVersion {
+	// an int of the major version, whereas Status.Responses.Cluster.PostgresVersion might be the ID
```

Comment:
```suggestion
	// an int of the major version, whereas Status.PostgresVersion might be the ID
```

Reply:
As of right now we only have MajorVersion on the status since that is what PGO cares about.